fix(worker): decreased time taken for VM creation to start

krsteen opened the pull request:
closes #329

krsteen committed:
refactor: reworked ResourceProvisioningService. Splitted the logic out and into separate utils

krsteen committed:
refactor: renamed ResourceProvisioningService_ to ResourceProvisioningService_OLD

krsteen committed:
refactor: renamed the new ResourceProvisioningService to that, it only had a temp name

krsteen committed:
refactor: renamed the new ResourceProvisioningService to that, it only had a temp name

krsteen committed:
style: fixed intellisense warnings

krsteen committed:
Merge branch 'fix/resource-creation-improvements' of https://github.com/equinor/sepes-api into fix/resource-creation-improvements

krsteen committed:
refactor: wip: new provisioning service

krsteen committed:
refactor: wip: improving resource creation

krsteen committed:
fix: (worker): fixed re-queue after max dequeue count exceeded

krsteen committed:
refactor: removed old resource provisioning service

HansKristianVikOlsen commented on src/Sepes.Functions/Sepes.Functions.csproj:
```diff
@@ -4,7 +4,7 @@
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
-    <NoWarn>1701;1702;IDE0060</NoWarn>
+    <NoWarn>1701;1702;IDE0060;IDE0063</NoWarn>
```
What warning is this?

krsteen replied:
"Using statement can be simplified", but when auto-fixing this problem, it removes the brackets around the using statement, and  breaks the flow of execution.

krsteen committed:
fix(sandbox): improved validation when moving to next phase

it was throwing false error about resources not being created, when they were in fact aborted

krsteen committed:
merged from develop
